webpack.production.config.js

```diff
@@ -31,18 +31,10 @@ module.exports = {
       filename: '[name]-[hash].min.css',
       allChunks: true,
     }),
-    new webpack.optimize.UglifyJsPlugin({
```

Comment:
Out of curiosity, why are we removing Uglify? I see that when running `npm run build`, the output JS is still minified, but I haven't figured out _where_ this is happening.

Reply:
The webpack version 4 production mode includes these plugins with their defaults:
```
Provides process.env.NODE_ENV with value production. Enables FlagDependencyUsagePlugin, FlagIncludedChunksPlugin, ModuleConcatenationPlugin, NoEmitOnErrorsPlugin, OccurrenceOrderPlugin, SideEffectsFlagPlugin and UglifyJsPlugin.
```
UglifyJsPlugin has a default to ignore IE8 support.